astropy/table/tests/test_pickle.py

```diff
@@ -72,11 +72,11 @@ def test_pickle_table(protocol):
         assert np.all(tp['d'] == t['d'])
         assert np.all(tp['e'].ra == t['e'].ra)
         assert np.all(tp['e'].dec == t['e'].dec)
-        assert type(tp['c']) == type(t['c'])
-        assert type(tp['d']) == type(t['d'])
-        assert type(tp['e']) == type(t['e'])
+        assert isinstance(tp['c'], type(t['c']))
```

Comment:
In all these tests, one does want to be sure unpickling gives exactly the same class, so replace `==` by `is`